packages/ui-lib/src/Filters/molecules/DatePicker.tsx

```diff
@@ -195,6 +195,29 @@ const CalHCell = styled(CalCell)`
   color: var(--grey-a11);
 `;
 
+const ContentDot = styled.div<{ hasContent: boolean, state?: CellStates, isToday?: boolean, }>`
+  position: absolute;
+  left: 18px;
+  bottom: 5px;
+  width: 4px;
+  height: 4px;
+  border-radius: 50%;
+  background-color: var(--info-10);
+
+  ${({ state }) => (state === 'single' || state === 'start' || state === 'end' || state === 'inside') && css`
+    background-color: var(--white-12);`
```

Comment:
For `inside` can we make this `--primary-12` please. The other states should remain `--white-12`, thank you.